feat(lcd_ili9341): Selectable default initial commands for LCD from three presets. (#687)
espressif/esp-bsp: components/lcd/esp_lcd_ili9341/README.md

```

There is an example in ESP-IDF with this LCD controller. Please follow this [link](https://github.com/espressif/esp-idf/tree/master/examples/peripherals/lcd/spi_lcd_touch).

## Predefined init vendor commands

> This driver contains some predefined vendor inititalization commands. It can help improve screen in some cases. Usage is in example below.

Reviewer: ```suggestion
This driver contains various predefined vendor initialization commands which in some cases can improve image quality. Usage is in example below.
```
Tiny nitpick but I think that it clarifies why these commands are useful.
